src/wp-includes/block-patterns.php

```diff
@@ -29,10 +30,9 @@ function _register_core_block_patterns_and_categories() {
 		);
 
 		foreach ( $core_block_patterns as $core_block_pattern ) {
-			register_block_pattern(
-				'core/' . $core_block_pattern,
-				require __DIR__ . '/block-patterns/' . $core_block_pattern . '.php'
-			);
+			$pattern           = require ABSPATH . WPINC . '/block-patterns/' . $core_block_pattern . '.php';
```

Comment:
I think the path was only changed in the gutenberg plugin because the plugin is in a different directory.

No need to bring this change over ton core.

Reply:
Thank you for explaining that 👍 

I did a brief check elsewhere in core and saw similar approaches in defining required paths, so mistakenly brought that over.